examples/all-clusters-app/esp32/README.md

```diff
@@ -18,28 +18,30 @@ CHIP.
 
 ## Supported Devices
 
-The CHIP demo application is intended to work on two categories of ESP32
-devices: the
+The CHIP demo application is intended to work on two categories of ESP32 devices
+and one categories ESP32C3 device: the
 [ESP32-DevKitC](https://www.espressif.com/en/products/hardware/esp32-devkitc/overview),
-and the [M5Stack](http://m5stack.com). On the [M5Stack](http://m5stack.com) this
-example displays a CHIP QRCode with the device's Soft-AP SSID encoded in the TLV
-section.
+the [M5Stack](http://m5stack.com) and the
+[ESP32C3-DevKitM](https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/hw-reference/esp32c3/user-guide-devkitm-1.html).
+On the [M5Stack](http://m5stack.com) this example displays a CHIP QRCode with
+the device's Soft-AP SSID encoded in the TLV section.
 
 ## Building the Example Application
 
 Building the example application requires the use of the Espressif ESP32 IoT
-Development Framework and the xtensa-esp32-elf toolchain.
+Development Framework and the xtensa-esp32-elf toolchain(for ESP32C3,the
+riscv-esp32-elf toolchain).
 
 The VSCode devcontainer has these components pre-installed, so you can skip this
 step. To install these components manually, follow these steps:
 
--   Clone the Espressif ESP-IDF and checkout release/v4.2 branch
+-   Clone the Espressif ESP-IDF and checkout v4.4-dev branch
 
           $ mkdir ${HOME}/tools
           $ cd ${HOME}/tools
           $ git clone https://github.com/espressif/esp-idf.git
           $ cd esp-idf
-          $ git checkout release/v4.2
+          $ git checkout v4.4-dev
```

Comment:
Let's try to use release/v4.3 which is the official release branch for ESP32-C3.